pkgs/development/compilers/ghc/8.6.5-binary.nix

```diff
@@ -38,8 +38,8 @@ stdenv.mkDerivation rec {
       sha256 = "1p2h29qghql19ajk755xa0yxkn85slbds8m9n5196ris743vkp8w";
     };
     x86_64-linux = {
-      url = "http://haskell.org/ghc/dist/${version}/ghc-${version}-x86_64-deb9-linux.tar.xz";
-      sha256 = "1pqlx6rdjs2110g0y1i9f8x18lmdizibjqd15f5xahcz39hgaxdw";
+      url = "http://haskell.org/ghc/dist/${version}/ghc-${version}-x86_64-fedora27-linux.tar.xz";
```

Comment:
Please add a comment why we use this one, just to prevent that somebody changes it back to `deb9` to fix some other issue (perhaps also linking https://github.com/NixOS/nixpkgs/issues/85924).